src/daft-dsl/src/lit.rs

```diff
@@ -108,6 +117,7 @@ impl Display for LiteralValue {
             Date(val) => write!(f, "{}", display_date32(*val)),
             Timestamp(val, tu, tz) => write!(f, "{}", display_timestamp(*val, tu, tz)),
             Float64(val) => write!(f, "{val:.1}"),
+            Series(series) => write!(f, "{}", series),
```

Comment:
How does this repr look like in python? I believe that this might be multiline and not look great in our expression repr.

Reply:
yep you're right, i made a separate [display function](https://github.com/Eventual-Inc/Daft/pull/1811/files#diff-9dd904ba457dac66f325cbb60a1e1e8028d377e9405db89efebc5e661ac66020R38-R51) for series literals, which will end up looking like: `lit([1, 2, 3])`. I could also make it more detailed, like: `lit(Series(int64): [1, 2, 3]), what do you think?